Yoast/YoastSEO.js
# Feature/german morphology

## Summary

This PR can be summarized in the following changelog entry:

* Adds word form recognition for German.

## Relevant technical choices:

* see the individual PRs for German nouns/adjectives/verbs:
   * Stemmer/nouns: https://github.com/Yoast/YoastSEO.js/pull/2054
   * Adjectives: https://github.com/Yoast/YoastSEO.js/pull/2133
   * Verbs: https://github.com/Yoast/YoastSEO.js/pull/2153
* For code review, just have a look at the [latest commit](https://github.com/Yoast/YoastSEO.js/pull/2171/commits/4d8ec759e173d3be1420f983b44f6bf04a31581c). The others have already been reviewed in the individual PRs (see above).

## Test instructions

This PR can be tested by following these steps:

* All functionality for morphology has been tested in the individual PRs; for this PR, a quick global check should be sufficient.
* Add a keyphrase with a noun, adjective and verb.
* Add different forms of the words in the text. For this, you can use a dictionary entry with forms, such as e.g. https://de.wiktionary.org/wiki/Flexion:kaufen.
* Make sure the different forms get recognized.

Fixes #2159
